controllers/reconcilers/kserve_authconfig_reconciler.go

```diff
@@ -18,35 +18,34 @@ package reconcilers
 import (
 	"context"
 	"fmt"
-
 	"github.com/go-logr/logr"
 	kservev1beta1 "github.com/kserve/kserve/pkg/apis/serving/v1beta1"
 	authorinov1beta2 "github.com/kuadrant/authorino/api/v1beta2"
+	"github.com/kuadrant/authorino/pkg/log"
```

Comment:
We are already using `github.com/go-logr/logr` library for logging. Why you are using new library?

Reply:
I think I tried to simplify logging at some point but that was a rabbit hole :) Left-over. Thanks for spotting! Fixed in 604162c